include/boost/buffers/flat_buffer.hpp

```diff
@@ -96,7 +96,7 @@ class flat_buffer
     std::size_t
     capacity() const noexcept
     {
-        return cap_ - in_pos_;
+        return cap_ - (in_pos_ + in_size_);
```

Comment:
This is correct for now, but it will change again once when we address https://github.com/cppalliance/buffers/issues/30.

Reply:
Hmm, maybe not, because `shrink_to_fit` would be a manual action and wouldn't happen automatically when calling `prepare`.

Reply:
We can implement `shrink_to_fit` and then after, decide if we want this behavior to be automatic in `prepare`.